### PR TITLE
feat: add import-service

### PR DESCRIPTION
**Pull request by spitaI**
### Task 5 (AWS Simple Storage Service - S3)

**Main tasks:**
✅ Task 5.1 completed

- `importProductsFile`: https://n5cx87r831.execute-api.eu-west-1.amazonaws.com/dev/import?name=test.csv

✅ Task 5.2 completed

Screen Shot 2020-11-15 at 16 42 23

✅ Frontend application is integrated with import service

- Link to PR in FE repo: https://github.com/spitaI/nodejs-aws-fe/pull/5

**Additional tasks:**
✅ async/await is used in lambda functions
✅ importProductsFile lambda is covered by unit tests
✅ At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder

**Done:** 14.11.2020 / deadline 15.11.2020
**Score:** 8/8

### CODE REVIEW

File: import-service/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          cors: true
+          request:
+            parameters:
+              querystrings:
+                name: true
+  importFileParser:
+    handler: handler.importFileParser
+    events:
+      - s3:
+          bucket: 'spitai-import'
```
Comment (petr150894) [inline import-service/serverless.yml:44]: It could be better if the bucket name will be stored in the environment variables. It won't affect the final mark, just an advice

File: import-service/src/handlers/__test__/handlers.test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import { S3_PUT_OPERATION } from '../../constants';
+import { getResponseObject, getS3ImportParams } from '../../utils';
+
+describe('AWS Lambda handlers tests', () => {
+  const testName = 'test.csv';
+  const getEventMock = name => ({
+    queryStringParameters: { name },
+  });
+
+  describe('importProductsFile handler tests', () => {
```
Comment (petr150894) [inline import-service/src/handlers/__test__/handlers.test.js:14]: no objections, just suggestion to add more tests for negative scenarios (400, 500 errors for example)

**Review by petr150894 — APPROVED** [review]
Well done!

File: import-service/src/handlers/__test__/handlers.test.js (hunk 2 above)
Comment (spitaI) [reply import-service/src/handlers/__test__/handlers.test.js:14]: Thank you for the review and your advices!
